graph/encoding/graph6/graph6_test.go

```diff
@@ -217,3 +218,46 @@ func linksTo(nodes ...graph.Node) map[int]struct{} {
 	}
 	return s
 }
+
+func TestLargeEncoding(t *testing.T) {
+	for _, l := range []int{
+		50, 60, 70, 80, 100, 1e4,
+	} {
+		g6 := Encode(implicitCycle(l))
+		if !IsValid(g6) {
+			t.Errorf("graph6-encoding unexpectedly invalid: %v", g6)
+		}
+		for i, b := range []byte(g6) {
+			if b < 63 || 126 < b {
+				t.Errorf("graph6-encoding contains invalid character at %d: %q", i, b)
+			}
+		}
```

Comment:
`numberOf()`  performs this check in `IsValid()`. Should `IsValid` and `numberOf` return additional errors for better reporting? Additional checks like these would then no longer need to be performed.

Reply:
No, I don't think so. The API as it is is nicely light, having additional things happening in the tests to support that is OK.